data/Landouda.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Landouda",
+    "displayStatsPublic" : true,    
```

Comment:
Also, we no longer need the `displayStatsPublic` key. So please remove this from your `json`.

```suggestion
```